src/client/components/layouts/Header/Header.test.tsx

```diff
@@ -230,11 +230,13 @@ describe('Header', () => {
 			jest.clearAllMocks();
 		});
 		describe('Par défaut', () => {
-			it('n‘affiche pas la navigation mobile', () => {
+			it('affiche la navigation mobile pour le burger menu quand le menu est fermer', () => {
 				mockUseRouter({ pathname: '/' });
 				render(<Header/>);
-				const menu = screen.queryByRole('navigation');
-				expect(menu).not.toBeInTheDocument();
+				const burgerMenu = screen.queryByRole('navigation', { name: 'ouvrir le menu principal' });
+				const menuPrincipal = screen.queryByRole('navigation', { name: 'menu principal' });
+				expect(burgerMenu).toBeInTheDocument();
```

Comment:
```suggestion
				expect(burgerMenu).toBeVisible();
```

On utilise plutôt `toBeVisible` désormais car certains éléments peuvent être présent dans le document mais invisible à l'utilisateur. 
(Bon c'est piegeux, toutes les autres assertions du fichiers sont en `toBeInTheDocument` parce qu'elle on été écrite il y a longtemps 😅)